src/opendr/engine/data.py

```diff
@@ -211,15 +211,32 @@ def __str__(self):
 class Image(Data):
     """
     A class used for representing image data.
+    OpenDR uses NCHW/RGB conventions
```

Comment:
Out of curiosity, what about grayscale images?

Reply:
Given that all implementations currently work with color images, I do not see any immediate need to support grayscale images. I can be done though quite easily by checking if image has only one channel